src/pyEQL/solution.py

```diff
@@ -2523,6 +2524,66 @@ def from_file(self, filename: str | Path) -> Solution:
             return Solution(**solution_dict)
         return loadfn(filename)
 
+        
+
+    @classmethod
+    def from_phreeqc(cls, path_to_pqi: str) -> "Solution":
+        """
+        Parse a PHREEQC .pqi input file and pull out only the SOLUTION block,
+        extracting solute amounts (assumed mol/L), plus pH, pE, and temperature if present.
```

Comment:
I believe the default units (if the user doesn't specify anything) are `mmol/kgw`, so at a minimum you should assume that instead of `mol/L`. But it would be even better to parse the `units` and `water` keywords as well so that this method can detect when users specify different units. See

https://wwwbrr.cr.usgs.gov/projects/GWC_coupled/phreeqc/html/final-56.html#89789

for a description of how they work.